lib/defaults.js

```diff
@@ -33,10 +33,10 @@ module.exports = {
     },
   ],
   groupTypeId: "sap.cds:service",
-  packages: function getPackageData (name, policyLevel,capNamespace) {
+  packages: function getPackageData (name, policyLevel,namespace) {
```

Comment:
When generating the name of package, we are now using `namespace`(taken from .cdsrc.json and package.json/name as fallback) instead of `capNamespace`(which is `csn.namespace` and can be undefined in case when the namespace is not given in service definition).

@Fannon @zongqichen : Do you agree?

Reply:
Maybe we should name things more clearly. Like this is the ORD namespace (not a CSN namespace)

```suggestion
  packages: function getPackageData (name, policyLevel, ordNamespace) {
```